scripts/byobu_startup.py

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import subprocess
+import sys
+import time
+
+from huntsman.pocs.utils import load_config
+
+WINDOWS = ["main-control",
+           "shutter-and-weather",
+           "camera-servers",
+           "camera-logs",
+           "dome-control",
+           "data managment"]
```

Comment:
If this is consistent with the others...

```suggestion
           "data-managment"]
```